test/inductor/test_cpu_repro.py

```diff
@@ -1960,6 +1960,24 @@ def _internal_check(
             with config.patch({"cpp.dynamic_threads": True}), set_num_threads(1):
                 _internal_check(fn, inps, "aten.scatter_reduce_")
 
+    @patch("torch.cuda.is_available", lambda: False)
+    @requires_vectorization
+    @torch._inductor.config.patch({"cpp.fallback_scatter_reduce_sum": False})
+    def test_scatter_using_atomic_add_vec(self):
```

Comment:
Since we have a special path for `num_threads == 1`, shall we have UTs covering both single-threaded and multi-threaded cases?

Reply:
Thanks for advices, added.